docs/sql-ref-datatypes.md

```diff
@@ -706,3 +708,53 @@ The following table shows the type names as well as aliases used in Spark SQL pa
 </table>
 </div>
 </div>
+
+### Floating Point Special Values
+
+Spark SQL supports several special floating point values in a case-insensitive manner:
+
+ * Inf/+Inf/Infinity/+Infinity: positive infinity
+   * ```FloatType```: 1.0f / 0.0f, which is equal to the value returned by <code>java.lang.Float.intBitsToFloat(0x7f800000)</code>.
```

Comment:
We don't need to describe the infinity behaivours?
 - infinity = infinity returns what?
 - In aggregations, what's the infinity behaivour?
 - ...